pronotepy/ent/__init__.py

```diff
@@ -1,2 +1,42 @@
-from .ent import *
-from .complex_ent import *
+from .ent import (
+    ac_clermont_ferrand,
+    ac_grenoble,
+    ac_lyon,
+    ac_orleans_tours,
+    ac_reims,
+    atrium_sud,
+    cas_agora06,
+    cas_agora06_educonnect,
+    cas_arsene76,
+    cas_cybercolleges42,
+    cas_ent27,
+    cas_kosmos,
+    cas_seinesaintdenis,
+    cas_seinesaintdenis_edu,
+    eclat_bfc,
+    ecollege_haute_garonne,
+    enc_hauts_de_seine,
+    ent2d_bordeaux,
+    ent77,
+    ent_auvergnerhonealpe,
+    ent_creuse,
+    ent_elyco,
+    ent_essonne,
+    ent_hdf,
+    ent_mayotte,
+    ent_somme,
+    ile_de_france,
+    l_normandie,
+    laclasse_educonnect,
+    laclasse_lyon,
+    lyceeconnecte_aquitaine,
+    monbureaunumerique,
+    neoconnect_guadeloupe,
+    occitanie_montpellier,
+    occitanie_montpellier_educonnect,
+    occitanie_toulouse,
+    ozecollege_yvelines,
+    paris_classe_numerique,
+    val_doise,
+)
+from .complex_ent import ac_rennes, ac_reunion
```

Comment:
Mypy doesn't like this. Maybe add `# type: ignore`, or remove it from the complex_ent file. I bet it's because of that.